documentation/peer-testing-1/peer-testing-report.md

```diff
@@ -27,18 +27,37 @@ In total, \_\_\_ individuals took part in our peer testing sessions. The members
 
 #### High
 
--
+- Timezone field does not "glow" red if the user did not enter anything, it only shows the text "please select your time-zone". (mentor and mentee form)
+- When signing in, if the user does not enter anything for email and password, it just says "please input something" for both fields, and then a pop-up that says "the email address is badly formatted" however both fields are blank. (login page)
+- From the above point, we should change "please input something" to "please input password" or "please input email", for the red text that pops up at the bottom of the field if the user does not enter an email address in the login page. (login page)
+- If the user enters a incorrect username and password, the website creates a popup that is "wordy" and maybe difficult to understand. (login page)
+- We should make different messages depending on of if the user did not enter any information in login screen (password/email) or if the user entered information but the information is wrong/does not have an account. (login page)
+- From above points, change the login pop up messages depending on what is and is not filled in. Change the red text that popups at the bottom of the field depending on what is and is not filled in. (login page)
```

Comment:
From reading these lines, I think they're more or less the same feedback (to change the red error text to be more descriptive, and modify the popups)? I might be misunderstanding though, but if not, can this section be condensed into one or two lines?

Reply:
I might of explained them badly, but they are supposed to be multiple things depending on the fields, like when information is wrong or not filled in then a popup will show, and then the red underneath the box when information is not filled in/wrong. So i was trying to explain that we need to fix both the popup (based on information that is filled but wrong, or information that is not filled in at all) and then we need to fix the read that shows around the box (based on information that is filled but wrong, or information that is not filled in at all). Sorry my explanations are so bad to describe what I mean

Reply:
No worries! Thanks for elaborating.